main.nf

```diff
@@ -46,21 +56,30 @@ process runAmber {
 
     script:
     """
-    amber \
-        -tumor ${tumor} \
-        -tumor_bam ${tumorBam} \
-        -output_dir \$PWD \
-        -threads ${params.cores} \
-        -loci ${params.loci} \
-        -ref_genome_version ${params.genomeVersion}
+    if [ -f "${params.outdir}/amber/${tumor}.amber.baf.tsv.gz" ] && \
+       [ -f "${params.outdir}/amber/${tumor}.amber.baf.pcf" ] && \
+       [ -f "${params.outdir}/amber/${tumor}.amber.qc" ]; then
```

Comment:
This is part of the updates for the force -- because the workflow has finished completely the runDir is already copied to the publishDir and it goes to redo these steps since the output files don't exist in the new runDir.